torch/_refs/__init__.py

```diff
@@ -1762,6 +1768,75 @@ def narrow(a: TensorLikeType, dim: int, start: int, length: int) -> TensorLikeTy
     return prims.slice_in_dim(a, start, start + length, axis=dim)
 
 
+def _normalize(a, norm_dims, eps):
+    computation_dtype = utils.get_computation_dtype(a.dtype)
+    a_acc = prims.convert_element_type(a, computation_dtype)
+    biased_var, mean = var_mean(a_acc, dim=norm_dims, unbiased=False, keepdim=True)
+    rstd = rsqrt(biased_var + eps)
```

Comment:
Can we just call `torch.rsqrt` here?

Reply:
My understanding is that the current thinking is to prefer calling `torch.foo` vs. `_refs.foo` if convenient, as it's fairly easy to map from `torch.foo` => `_refs.foo` but not the other way around.

See @mruberry's comment here: https://github.com/pytorch/pytorch/pull/78689#discussion_r889159026

Reply:
It's very easy to map `_refs.rsqrt` to `torch.rsqrt` because `_refs.rsqrt` is a wrapper of `torch.rsqrt` for regular and meta tensors and `test_decompy.py` passing indicates that it's fine as it is.

I changed this to `torch.rsqrt`.

Reply:
From the perspective of the "correctness" tests it's fine. But, if I understand correctly, `_refs.rsqrt => prims.rsqrt`, and then you're saying that `prims.rsqrt` simply calls `torch.rsqrt` under the hood. 

But when we trace things out, using `_refs.rsqrt` doesn't have an interception point to allow us to decompose `native_layer_norm` but not decompose `_refs.rsqrt`.

TBH, for `rsqrt` this is a somewhat strange discussion haha since `_refs.rsqrt == prims.rsqrt == aten::rsqrt`. But it matters more for `var_mean`.

For example, say that I'm a compiler that wants `native_layer_norm` decomposed, but I have a special handling for `var_mean`. If we call `_refs.var_mean`, then our trace has no option to trace out `var_mean` - we'll get `var` followed by `mean`.

If we use `torch.var_mean` though, then we'll have the choice to either further decompose into `var` and `mean` or preserve `var_mean` in our trace.

Reply:
Because of the way the current context mapping is implemented it's a one-way torch. -> refs. transform, so we should get in the habit of preferring torch. calls when possible. In some cases the torch operation doesn't have as much functionality as the ref, and in cases where that extra functionality is used the ref should be called explicitly, instead.